OneMore/Commands/File/Markdown/MarkdownConverter.cs

```diff
@@ -158,21 +195,43 @@ public MarkdownConverter RewriteTodo(IEnumerable<XElement> paragraphs)
 				{
 					var cdata = run.GetCData();
 					var wrapper = cdata.GetWrapper();
+					if (wrapper.FirstNode is XText)
+					{
+						cdata.Value = wrapper.GetInnerXml();
+					}
+					while (wrapper.FirstNode is not XText && wrapper.FirstNode is not null)
```

Comment:
Ostensibly the same as wrapper.DescendantNodes().OfType<XText>().First().... yes?